packages/Molecules/Pagination/index.vue

```diff
@@ -1,9 +1,17 @@
 <template>
-  <div class="rpl-pagination">
+  <nav class="rpl-pagination" role="navigation" :aria-label="label || 'search'">
```

Comment:
Role of Navigation is implied by the element `<nav>`.
https://www.w3.org/TR/html5/sections.html#the-nav-element
recommended not to set:
> Allowed ARIA role attribute values:
>     navigation role (default - do not set). 
> In the majority of cases setting an ARIA role and/or aria-* attribute that matches the default implicit ARIA semantics is unnecessary and not recommended as these properties are already set by the browser.

Reply:
Ah this is one of those ideal world vs real world things - https://www.w3.org/WAI/GL/wiki/Using_HTML5_nav_element says : 

> "However, in order to make the structure more accessible to user agents that support ARIA as well as ensuring that user agents that don't support HTML5 can also understand the structure, adding the ARIA role="navigation" is advised."

I'm not sure if that is still relevant, but it was about 2 years ago when tested against a range of screen readers.

Will remove it I think, because some automated tests balk at it.